### PR TITLE
Release 1

### PR DESCRIPTION
**Pull request by keroloussamy**
In this PR, I made the following
1- Add all initial files.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  color: rgb(92, 89, 89);
+}
+
+#todo-list li i {
+  margin-left: auto;
+}
+
+.add-item input {
+  border: 0;
+  width: 100%;
```
Comment (Mre55) [inline src/style.css:28]: You can define the width 100% once and re-use it below to make your code DRY.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  if (!JSON.parse(localStorage.getItem('todo-list'))) {
+    LSOperations.storeList(LSOperations.setSortingIndex(todoList));
+  }
+
+  manipulateHtml.displayList();
+
+  const checkboxes = document.querySelectorAll('.checkbox');
+  [...checkboxes].forEach((button) => {
+    button.addEventListener('change', LSOperations.completeTask);
+  });
```
Comment (Mre55) [inline src/index.js:17]: On the live demo when the user checks the checkbox to complete the task the input (task) should have to be crossed and its opacity has to decrease as in the template.

File: src/localStorageOperations.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+  const completeTask = (event) => {
+    const list = getList();
+    if (event.target.checked === true) {
+      list[event.target.id].completed = true;
+    } else {
+      list[event.target.id].completed = false;
+    }
+    storeList(list);
+  };
```
Comment (Mre55) [inline src/localStorageOperations.js:21]: Here you can add the line cross when the task is completed and when the user unchecks the checkbox to make the task undone it should return to the default state.

**Review by Mre55 — APPROVED** [review]
## CHANGES REQUIRED 🔴 🛑
Hello, @keroloussamy great job you are doing so far in this project. 💯

## TO HIGHLIGHT:  🎉

- Your project is professional ✔️
- Your repo is professional ✔️
- Well structured files ✔️
- Correct Github flow ✔️
- Linter checked ✔️

However, there are a few changes to make to move to the next project 🛠️ . Keep working hard 💪 💻

## Required Changes ♻️

Complete the rest of the sections
Please make these required changes. Check the below comments and send it again for another review 👍 💪
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## HAPPY CODING!!! 😄
You have done a good job so far 🚀

File: src/style.css (hunk 1 above)
Comment (keroloussamy) [reply src/style.css:28]: Done

File: src/index.js (hunk 2 above)
Comment (keroloussamy) [reply src/index.js:17]: Done

File: src/localStorageOperations.js (hunk 3 above)
Comment (keroloussamy) [reply src/localStorageOperations.js:21]: Done

**Comment by keroloussamy** [comment]
Thanks, @Mre55 for your review.

**Pull request merged**